scripts/wercker/run-rainforest

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -o errexit
+
+declare hostname=$(head --lines 1 $1 | awk '{print $3}')
+declare revision=$(git rev-parse --short HEAD)
+
+for file in $(ls -1 tests/spec/rainforest/*.rfml); do
```

Comment:
issue: 
   we will be missing some of the files with `ls -1 tests/spec/rainforest/*.rfml`

folder: 
   tests/spec/rainforest/account/login.rfml
  login.rfml is a file in account `folder`